build.gradle

```diff
@@ -50,11 +51,14 @@ dependencies {
 
     // Needed for build and run
     implementation "org.slf4j:slf4j-api:${slf4jVersion}"
+    implementation "org.apache.httpcomponents:httpclient:${httpclientVersion}"
+    implementation "org.odpi.egeria:topic-integrator-api:${egeriaversion}"
```

Comment:
this should be 'compileOnly' - for all of the components we think are already in the chassis. It's more specific, and would avoid the libraries being picked up to include in any uber jar